vision/cdcgan_mnist/cGAN_mnist.jl

```diff
@@ -75,6 +76,8 @@ function (m::Generator)(x, y)
 end
 
 function load_data(hparams)
+    MLDatasets.MNIST.download(i_accept_the_terms_of_use=true)
```

Comment:
let's remove this. It's not needed when including the script from a REPL (julia's typical usage)

Reply:
Will do. As someone who ran this using the VSCode extension, it just printed out the text and there was no way for me to respond that "yes please download it". I suspect that anyone who doesn't use the REPL by default (mostly newer users) could stumble in the same spot. Should I add a comment?

Reply:
That probably also explains what `cd(@__DIR__)` is doing in all these files :sweat_smile: excuse my ignorance